.github/workflows/periodic-pytorch-libtorch-linux-xenial-cuda11.3-cudnn8-py3.6-gcc7.yml

```diff
@@ -98,7 +99,7 @@ jobs:
 
   build:
     runs-on: linux.2xlarge
-    needs: calculate-docker-image
+    needs: [calculate-docker-image, ]
```

Comment:
is this syntax ok? I'm not a yaml expert

Reply:
yes, trailing comma is allowed in yaml